components/next-event-card/NextEventCard.vue

```diff
@@ -3,13 +3,25 @@
     :href="`/events/${event.slug}`"
     class="next-event-card"
   >
-    <span class="next-event-card__title h3">
-      {{ event.title }}
-    </span>
+    <div class="next-event-card__content">
+      <img
+        v-if="event.bannerIcon"
+        alt=""
+        :src="`${event.bannerIcon.imgixHost}/${event.bannerIcon.value.path}`"
```

Comment:
**Question**: should this URL have transformation parameters for compression, quality, width etc?

Reply:
Yes, it should, will fix it!